lua/chatcommands/coinflip.lua

```diff
@@ -0,0 +1,153 @@
+CoinFlips = CoinFlips or {}
+
+RegisterChatCommand({'coin','coinflip'},function(ply,arg)
+	local t = string.Explode(" ",arg)
+	local p = tonumber(table.remove(t))
+	if #t == 1 and t[1]:lower() == "accept" then checkCoinFlipRequest(ply, p) return end
+
+	if p == nil then
+		if #t == 1 and t[1]:lower() == "accept" then
+			ply:ChatPrint("[orange]!coinflip accept points")
+		else
+			ply:ChatPrint("[orange]!coinflip player points")
+		end
+	else
+		local to,c = PlyCount(string.Implode(" ",t))
+		if c == 1 then
+			if ply == to then
+				ply:ChatPrint("[red]You can't coinflip yourself!")
+			elseif hasCoinflipRequest(to) then
+				ply:ChatPrint("[red]That player currently has a coinflip request active!")
+			elseif p >= 1000 then -- minimum 1000 point coinflip
+				initCoinFlip(ply,to,p)
+			else
+				ply:ChatPrint("[red]A coin flip must be a minimum of 1000 points. No pussy bets!")
+			end
+		else
+			ply:ChatPrint("[red]Player "..string.Implode(" ",t).." not found")
+		end
+	end
+end,{global=true,throttle=true})
+
+timer.Create( "CoinFlip", 1, 0,
+function()
+	local usedID = nil
```

Comment:
Make a list of things to remove then iterate that list removing things.

Alternately, make a new empty table and add everything you aren't going to remove to it, then set CoinFlips = (the new table) when you're done.